tests/gold_tests/pluginTest/prefetch/prefetch_bignum.test.py

```diff
@@ -71,7 +71,15 @@
     f'curl --verbose --proxy 127.0.0.1:{ts.Variables.port} http://domain.in/texts/demo-3842948374928374982374982374.txt')
 tr.Processes.Default.ReturnCode = 0
 
+tr = Test.AddTestRun('Await transactions to finish logging.')
+p = tr.Processes.Default
+p.Command = 'echo waiting for transactions to finish logging'
+await_process = tr.Processes.Process('await', 'sleep 60')
+await_process.Ready = When.FileContains(ts.Disk.traffic_out.Name, '3842948374928374982374982377')
+await_process.StartupTimeout = 30
```

Comment:
Should lines 74-79 be a function where you pass in tr and the string for FileContains?

Reply:
That's a good idea.

Reply:
I added a `Test` extension to do this. That way any autests can re-use this implementation.

Thanks for suggesting breaking this out into a reusable module.